tests/playwright/assets/breakpoints.ts

```diff
@@ -44,4 +44,18 @@ export default class {
 
 		await this.page.waitForSelector( '#elementor-editor-wrapper' );
 	}
+
+	async resetBreakpoints() {
+		await this.page.click( '#elementor-panel-footer-responsive' );
+		await this.page.click( '#e-responsive-bar__settings-button' );
+		await this.page.waitForSelector( 'text=Active Breakpoints' );
+
+		const removeBreakpointButton = '#elementor-kit-panel-content .select2-selection__choice__remove';
+		while ( await this.page.$( removeBreakpointButton ) ) {
```

Comment:
`$` is [deprecated](elementor-panel-footer-responsive)
```suggestion
		while ( await this.page.locator( removeBreakpointButton ) ) {
```

Reply:
Fixed in https://github.com/elementor/elementor/pull/27617/commits/ba404aa8d49a71f63e0279353aac83b8961b6182